package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --env=jsdom-fourteen",
```

Comment:
Will be removed when `react-scripts` v4 is released, as it will include Jest v26, which includes a newer version of JSDOM, which includes a shim of `MutationObserver`, which is required by `testing-library`.........